### PR TITLE
Add HLR request feature

### PR DESCRIPTION
**Pull request by Earendil95**
Realize [this](https://smsaero.ru/api/description/#hlr) feature

### CODE REVIEW

File: lib/sms_aero/operations/hlr.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class SmsAero
+  HLR_STATUSES = {
+    1 => :available,
+    2 => :unavailable,
+    3 => :nonexistent
+  }
```
Comment (nepalez) [inline lib/sms_aero/operations/hlr.rb]: Could you please `.freeze` a constant here?

File: lib/sms_aero/operations/hlr.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  operation :hlr_status do
+    documentation "https://smsaero.ru/api/description/#hlr"
+
+    path { "hlrStatus" }
+
+    query { attribute :id, Types::Coercible::String.constrained(filled: true) }
+
+    response :success, 200, format: :json, model: Answer do
+      status_proc = proc { |s| HLR_STATUSES[s.to_i] }
+      attribute :status, status_proc
```
Comment (nepalez) [inline lib/sms_aero/operations/hlr.rb]: I don't think `status_proc` needs its own line; one line provides better readability here:

```ruby
attribute :status, ->(s) {  HLR_STATUSES[s.to_i] }
```

**Comment by Earendil95** [comment]
Hmm, I wanted to open one more PR to master with respect to new architecture in v0.1.x
And form this I want release smth like 0.0.10

**Comment by nepalez** [comment]
@Earendil95 yeah, I’ve already added both 0.0.10 and 0.1.2 with the same functionality but slightly different implementations

**Pull request merged**